gateway/1.8/modules/ROOT/pages/flex-architecture-high-availability.adoc

```diff
@@ -0,0 +1,94 @@
+
+
+[[implementation-1b]]
+=== Implementation 1B: Region-Specific Flex Gateway
+
+In this implementation:
+
+* Each region has a distinct Flex Gateway with all resources as independent entities.
+* A separate Flex Gateway is registered for each region.
+* Logs and metrics are specific to their region because each Flex Gateway is a distinct entity in Anypoint Platform.
+* The DNS service is configured for active-active failover to direct traffic to any region at all times.
+* API deployments are specific to each Flex Gateway. For each intended API a region supports, you must deploy an API instance to the region's Flex Gateway. 
+* API Groups can share contracts (access requests) across regions.
+* Shared storage does not require synchronization. 
+* There is no cross-regional traffic.
+
+In the following diagram, all APIs are duplicated and do not require identical configurations:
+
+image:Flex-Gateway-DR-2.png["A detailed view of implementation 1B, which contains the necessary services to support High Availability in region-specific Flex Gateways."]
+
+[[implementation-2]]
+== Implementation 2: Disaster Recovery for APIs
+
+This implementation is similar to <<implementation-1a>>. However, the DNS service is active-passive, so the standby region only receives traffic if the primary region becomes unhealthy. This implementation is less complex and cheaper to configure than implementation 1a.
+
+Redis replication is not required if a cache loss during failover is acceptable.
+
+In the following diagram, Flex Replicas in different availability zones have distinct Kubernetes deployments, and Region B provides high availability:
+
+image:Flex-Gateway-DR-3.png["A detailed view of implementation 2, which contains the necessary services to enable Disaster Recovery for APIs on Flex Gateway."]
+
+[[implementation-3]]
+== Implementation 3: Direct Traffic to the Closest Region
+
+In both <<implementation-3a>> and <<implementation-3b>>, Flex Gateway directs traffic to the most convenient region to reduce latency. Choose which implementation dependent on your policy requirements.
+
+[[implementation-3a]]
+=== Implementation 3A: Direct Traffic to APIs With the Same Policies
+
+This implementation is similar to <<implementation-1a>> and has the same considerations. However, Redis replication is not required because requests are sent to specific regions. Configure high availability or disaster recovery zones in each region to ensure API availability:
+
+image:Flex-Gateway-DR-4.png["A detailed view of implementation 3A, which contains the necessary services to enable direct customer requests to the closest region."]
+
+To configure the same API with the same policies to two regions, use multiple upstreams to direct the traffic to the most convenient region. To learn more about multiple upstreams, see:
```

Comment:
I don't think Flex multiple upstreams is at play here. 
Each region is a different upstream to the DNS resolver (e.g. R53) but once traffic lands in Flex, the upstreams are all in-region